tests/test_series.py

```diff
@@ -1287,9 +1335,15 @@ def test_series_min_max_sub_axis() -> None:
     sm = s1 * s2
     sd = s1 / s2
     check(assert_type(sa, pd.Series), pd.Series)
-    check(assert_type(ss, pd.Series), pd.Series)
+    check(
+        assert_type(ss, pd.Series),  # pyright: ignore[reportAssertTypeFailure]
```

Comment:
@Dr-Irv only thing that I am blocked on right now is these ones, they still get mapped to `Series[int]` although the two elements are `Series[Unknown]`